src/main/java/de/dhbw/karlsruhe/domain/services/SettingService.java

```diff
@@ -0,0 +1,22 @@
+package de.dhbw.karlsruhe.domain.services;
+
+import de.dhbw.karlsruhe.domain.models.GameInformation;
+import de.dhbw.karlsruhe.domain.models.Setting;
+
+public class SettingService {
+
+    private final UserService userService = DependencyFactory.getInstance().getDependency(UserService.class);
+
+    public void toggleFieldValidation(Setting setting) {
+        setting.setFieldValidation(!setting.getFieldValidation());
+    }
+
+    public void toggleValueHint(Setting setting) {
+        setting.setValueHint(!setting.getValueHint());
+    }
+
+    public Setting getSetting() {
```

Comment:
Maybe rename to "getSettingOfCurrentUser" to make it clear, where the value ist coming from and to make it different from the getSetting() call of the User class.